src/reanimated2/testUtils.js

```diff
@@ -0,0 +1,18 @@
+import hash from 'string-hash-64';
+
+export function createWorkletMock(fun) {
+  var _f = function _f() {
+    return fun();
+  };
+
+  var funString = fun.toString();
+
+  _f._closure = {};
+  _f.asString = funString;
+  _f.__workletHash = hash(funString);
+  _f.__location = 'jest tests fixture (1:0)';
+
+  global.__reanimatedWorkletInit(_f);
+
+  return _f;
+}
```

Comment:
I noticed there is `createWorklet` function (that is listed among other function to be processed by babel plugin) so I decided to create test stub for this one.

In longer term, there are two options:
- enable babel transform in tests (are there any limitations against?)
- if babel tranform for tests is not an option, split `Hooks.js` and other functions (that are transformet by the plugin) into separate files. Then create hook mock that will create worklet mock and pass it to original hook implementation.
eg:
```js
function useWhatever(callback) {
   const worklet = createWorkletMock(callback)
   return useWhateverOriginal(worklet)
}
```

Reply:
> enable babel transform in tests (are there any limitations against?)

Depends if you're talking about Reanimated2 internal tests or enabling transforms by every user of reanimated. AFAIK they're disabled by default because transforming node_modules take time.

I think @karol-bisztyga or @Szymon20000 can decide on this one.

Reply:
> Depends if you're talking about Reanimated2 internal tests or enabling transforms by every user of reanimated. 

Enabling transform for internal tests.

> AFAIK they're disabled by default because transforming node_modules take time.

Was under impression node_modules is excluded from transformation run.